subprojects/common/elastic/src/integTest/kotlin/com/avito/android/elastic/ElasticLogIntegrationTest.kt

```diff
@@ -0,0 +1,29 @@
+package com.avito.android.elastic
+
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+
+internal class ElasticLogIntegrationTest {
+
+    @Disabled
```

Comment:
Why Disabled?

Reply:
i will remove this test; don't know how to properly assert things here